buildSrc/shared-dependencies.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     implementation(libs.androidToolsRepository) // com.android.repository for Stable AIDL plugin
     implementation(libs.androidToolsSdkCommon) // com.android.ide.common for Stable AIDL plugin
     implementation(libs.kotlinGradlePluginz)
+    implementation(libs.kotlinBinaryCompatibilityValidatorGradlePluginz)
```

Comment:
```suggestion
    implementation(libs.kotlinBinaryCompatibilityValidatorGradlePlugin)
```

Reply:
This was confusing name, when I was writing it :) (I keep the pattern of other plugins). But let's change it.